src/main/java/core/basesyntax/MyHashMap.java

```diff
@@ -1,19 +1,94 @@
 package core.basesyntax;
 
 public class MyHashMap<K, V> implements MyMap<K, V> {
+    private static final int DEFAULT_CAPACITY = 16;
+    private static final int DEFAULT_GROW_FACTOR = 2;
+    private static final double DEFAULT_LOAD_FACTOR = 0.75;
+    private Node<K,V>[] table;
```

Comment:
I would add an empty line here between static and other private

Reply:
I think it can be read well as constants are much longer than other fields =)